pkgs/development/tools/misc/texinfo/4.13a.nix

```diff
@@ -12,12 +19,14 @@ stdenv.mkDerivation rec {
   buildInputs = [ ncurses ];
   nativeBuildInputs = [ xz ];
 
-  env.NIX_CFLAGS_COMPILE = toString (lib.optionals stdenv.cc.isClang [
-    "-Wno-error=implicit-function-declaration"
-  ]);
+  env.NIX_CFLAGS_COMPILE = toString (
+    lib.optionals stdenv.cc.isClang [ "-Wno-error=implicit-function-declaration" ]
```

Comment:
What is a point? The resulting string became longer.

Reply:
I ran the files through [`nixfmt`](https://github.com/NixOS/nixfmt); a separate commit might have been a better choice…

Reply:
https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#code-conventions